debian/changelog

```diff
@@ -1,3 +1,53 @@
+ubuntu-advantage-tools (27.13~23.04.1) lunar; urgency=medium
+
+  * d/bash-completion:
+    - enable autocomplete for the 'pro' command (GH: #2280)
+  * d/control:
+    - update the package description
+  * d/postinst:
+    - remove unauthenticated esm repos from Xenial systems (LP: #1990378)
+  * New upstream release 27.13 (LP: #2003018)
+    - apt:
+      + remove logic which added repositories and pinned them to 'never' to
+        enable access to esm package lists
+      + add functionality to create and update a local apt esm cache with
+        the lists for esm-infra and esm-apps
+    - apt-hook: update the cpp hook to use the local esm apt cache
+    - apt-news:
+      + fetch and display APT News in apt upgrade
+      + show contract expiration notices in the apt news output
+    - attach: support attaching without being able to install snapd
+      (LP: #1997514)
+    - cli:
+      + do not show invalid subcommands in autocomplete (GH: #2279)
+      + add commands to support the magic-attach flow
+    - config: add apt_news_url option
+    - docs: reorganize documentation and correct information
```

Comment:
I think we can remove this changelog entry

Reply:
I kept it there for the same reason we kept it before: it is a significant amount of commits docs-related, and Sally's work is focused here, so I'd like to report it